data/src/main/java/com/microsoft/azure/kusto/data/KustoResultSetTable.java

```diff
@@ -139,7 +139,7 @@ protected KustoResultSetTable(JSONObject jsonTable) throws KustoServiceQueryErro
         return currentRow;
     }
 
-    @Override
+
```

Comment:
I think the file needs re-formatting
Sometimes the `@Override` is replaced with a blank line, other times it doesn't
Make sure the space between methods stays consistent